src/main/scala/com/microsoft/hyperspace/actions/RefreshDeleteAction.scala

```diff
@@ -89,37 +70,4 @@ class RefreshDeleteAction(
       previousIndexLogEntry.numBuckets,
       indexConfig.indexedColumns)
   }
-
-  /**
-   * Compare list of source data files from previous IndexLogEntry to list
-   * of current source data files, validate fileInfo for existing files and
-   * identify deleted source data files.
-   */
-  private lazy val deletedFiles: Seq[String] = {
```

Comment:
**To Reviewers:** `deletedFiles` is moved to (new) class `RefreshDeleteActionBase`.